index.html

```diff
@@ -2467,154 +2467,153 @@ <h3>The <dfn>NFCReaderOptions</dfn> dictionary</h3>
-                    Set bit 7 (most significant bit) to the value 0
+                    Set bit `7` (<a>MB field</a>) to the value 0
                     (meaning <a>UTF-8 encoding</a>).
                   </li>
                   <li>
-                    Set bit 6 to the value 0 (reserved).
+                    Set bit `6` (<a>ME field</a>) to the value 0 (reserved).
                   </li>
                   <li>
-                    Let <var>offset</var> be the length of the
-                    <var>language</var> <a>string</a>.
+                    Let |offset| be the length of the
+                    |language| <a>string</a>.
                   </li>
                   <li>
-                    Set bit 5 to bit 0 to <var>offset</var>.
+                    Set bit `5` to bit `0` to |offset|.
                   </li>
                 </ol>
               </li>
               <li>
-                Set position 1 to <var>offset</var> + 1, inclusive, of the <a>byte sequence</a>
-                of <var>ndef</var>'s <a>PAYLOAD field</a> to the result of running <a>ASCII encode</a>
-                on <var>language</var>.
+                Set position 1 to |offset| + `1`, inclusive, of the <a>byte sequence</a>
+                of |ndef|'s <a>PAYLOAD field</a> to the result of running <a>ASCII encode</a>
+                on |language|.
               </li>
               <li>
-                Set position from <var>offset</var> + 1 to end, of the <a>byte sequence</a>
-                of <var>ndef</var>'s <a>PAYLOAD field</a> to the result of running <a>UTF-8 encode</a>
-                on <var>record</var>'s data.
+                Set position from |offset| + `1` to end, of the <a>byte sequence</a>
+                of |ndef|'s <a>PAYLOAD field</a> to the result of running <a>UTF-8 encode</a>
+                on |record|'s data.
               </li>
             </ol>
           </li>
           <li>
-            Return <var>ndef</var>.
+            Return |ndef|.
           </li>
         </ol>
       </p>
       </section>
 
       <section><h3>Mapping URL to NDEF</h3>
       <p>
-        To <dfn>map a URL to NDEF</dfn> given a <var>record</var>, run these
+        To <dfn>map a URL to NDEF</dfn> given a |record:NDEFRecordInit|, run these
         steps:
         <ol>
           <li>
-            If <var>record</var>'s data is not a <a>string</a>,
+            If |record|'s data is not a <a>string</a>,
             [= exception/throw =] a {{"TypeError"}} and abort these steps.
           </li>
           <li>
-            Let <var>urlRecord</var> be the result of
-            <a data-lt="url parser">parsing</a> <var>record</var>'s data.
+            Let |urlRecord:string| be the result of
+            <a data-lt="url parser">parsing</a> |record|'s data.
           </li>
           <li>
-            If <var>urlRecord</var> is failure, [= exception/throw =] a
+            If |urlRecord| is failure, [= exception/throw =] a
```

Comment:
SyntaxError seems like an odd choice to me...

Reply:
I think it's just a TypeError.

Reply:
(See also what the URL spec does... it seems to also throw TypeError)

Reply:
OK, I will file a separated issue.